app/src/components/token/Token.tsx

```diff
@@ -43,11 +43,11 @@ const tokenBaseCSS = css`
   align-items: center;
   gap: var(--ac-global-dimension-static-size-100);
   font-size: var(--ac-global-dimension-static-font-size-75);
+  font-weight: 400;
   line-height: var(--ac-global-line-height-s);
   padding: 0 var(--ac-global-dimension-static-size-100);
   border-radius: var(--ac-global-rounding-large);
-  border: 1px solid
-    lch(from var(--ac-internal-token-color) calc((l) * infinity) c h / 0.3);
+  border: 1px solid var(--ac-global-color-grey-400);
```

Comment:
let's not change the token colors in this PR.

Reply:
I personally quite dislike this
<img width="147" height="165" alt="Screenshot 2025-08-06 at 9 32 54 AM" src="https://github.com/user-attachments/assets/f4538d29-0389-46d3-a496-5baecadf8636" />